### ISSUE TITLE
Inconsistent options under Look&Feel > [Portrait] or [Landscape]

### ISSUE THREAD

**Issue by luckkmaxx** [Slion/Fulguris#427]
# Description
This options are not clear, but so confusing. Some options may be reworked and documented.

# Screenshots
![Screenshot_20230117-235028_Fulguris](https://user-images.githubusercontent.com/14180594/213034641-7c669020-222d-4dac-bcda-c08ae0fb2b6b.jpg)

# Screenshot explanation
1) My red text compares both TOOLBAR options. The last one seems to do nothing and says basically the opposite of the first one (I had to force app in english to be sure there wasnt a translation issue).

2) The green text 😓: (maybe I had to set this on red LOL)
In this case first and second options are very confusing too.
What I understand is the first one is to switch between classic tabs (like on PC) and a... vertical list of tabs? In this case:
  2.a) The result when enabled is that the webpage is placed on the side meant for the tab list:
![Screenshot_20230117-234750_Fulguris](https://user-images.githubusercontent.com/14180594/213036081-c452967f-ce59-4eea-827d-fe17d590a9ac.png)
  2.b) Both (enabled or disabled) cant be compatible with the next option wich wraps all tabs inside a popup window. So maybe should be: As first option set "Tab bar in drawer", wich if is enabled then the second option "Vertical tab bar" should be forbiden to set due to incompatibility (and maybe change the name to "List of tabs on left side\n disable it to see classic tabs").

3) Classic tabs on portrait cant show the + button, nor sessions or any related button.

I hope Im explaining well, and not forget anything to tell.

**Comment by luckkmaxx** [comment]
May be this issue could be divided on other issues for each problem. Feel free to do it if you consider.

**Comment by Slion** [comment]
"Show toolbar at the top of the page" is only relevant if you are using options to hide the toolbar. It will make sure that the toolbar is shown when you scroll back to the top of the page though the toolbar could still be displayed at the bottom 😁

**Comment by Slion** [comment]
I agree some of those options can be quite confusing, it's the price to pay for flexibility. Though things could certainly be improved both in code and in presentation. A setup wizard would help too.

**Comment by Slion** [comment]
"Toolbars at the bottom" does not apply just to the URL bar. It's applies to various toolbars across the UI. In the sessions menu and in the tab draw for instance.

**Comment by Slion** [comment]
Some combination of options might not makes sense on your device depending of you configuration. Just don't use them then, them may be useful on other devices with different configuration.

**Comment by Helium314** [comment]
What could be done is disabling (or hiding) some settings that don't make sense, e.g. disable "show tool bar when scrolling up" when "hide tool bar when scrolling down" is switched off.
This would also provide some implicit information that those settings are connected.
